### ISSUE TITLE
create multiclass classifier

### ISSUE THREAD

**Issue by bogsnork** [bogsnork/nectarvision#4]

**Comment by bogsnork** [comment]
commits: 
8fcaafd008e06a45368603d0593a85540c6a5faa
c83cd525038e986acf216cc06f3ccad222615c05
03b35ba8fe55cf35a0d8d1df781c875ef51b542b
11c408bd31a4940a4809c761d6934f6ebb867bf9
7ca8700c55592f2b3c490ea8883f83923353fefb
0d21e465a78588d4ef92e428bcd698bd1688922f
72755be3ae249b15e3e7e62105d5a2857ef57349
6ec045c27337efa50930bc5f189f4df71841e858
6ec045c27337efa50930bc5f189f4df71841e858

**Comment by bogsnork** [comment]
Classifier works.  Fitted with 9 epochs.  Binary crossentropy validation loss = 0.15; validation accuracy = 0.951.  But looking at a few examples, classification is not very good.  Possibly because true-negatives are being overwhelming accuracy, or maybe because of class imbalance.  Ideally need confusion matrix and improve metrics for specificity.

**Comment by bogsnork** [comment]
![image](https://user-images.githubusercontent.com/23337510/70073661-380e8a00-15f1-11ea-8183-c5117982f000.png)
![image](https://user-images.githubusercontent.com/23337510/70073723-52e0fe80-15f1-11ea-86b3-edf998b11887.png)

**Comment by bogsnork** [comment]
closing for now

**Comment by bogsnork** [comment]
closing and will solve classification problems when developing localisation model
